### PR TITLE
Added replaceAll polyfill

### PR DESCRIPTION
**Pull request by informathemusic**

### CODE REVIEW

**Comment by daanbreur** [comment]
what was this polyfill usefull for?

**Comment by daanbreur** [comment]
i dont see the point

**Comment by daanbreur** [comment]
i see... sorry for declining it in the first place... 
If you dont mind i copy the code and put it in again ;)
a lot of browsers actually dont do support it according to [https://caniuse.com/#search=replaceAll](https://caniuse.com/#search=replaceAll)

**Comment by informathemusic** [comment]
You asked me to answer here, well here an answer :)

**Comment by daanbreur** [comment]
> i see... sorry for declining it the first time...
> If you dont mind i copy the code and put it in again ;)
> a lot of browsers actually dont do support it according to https://caniuse.com/#search=replaceAll

answer this

**Comment by informathemusic** [comment]
Well i don't mind

File: web/chatblocks.js [hunk 1]
```diff
@@ -134,6 +134,10 @@ function codeUpdateHandler( event ) {
 	<body>
 		<script type="text/javascript">
 		try {
+			//Will only work if the first arg is not a regex
```
Comment (daanbreur) [inline web/chatblocks.js:137]: can you remove this line xD

**Comment by daanbreur** [comment]
can you add a check for if it already exists before you overwrite
???

**Comment by informathemusic** [comment]
Sure!

**Comment by informathemusic** [comment]
Done

**Comment by daanbreur** [comment]
dont think thats actually code tht works

**Comment by daanbreur** [comment]
can you do it with a normal if

**Comment by informathemusic** [comment]
I can, but i'm heading to bed rn. And this is working code

**Comment by daanbreur** [comment]
doesnt work for me
it overwrites while the browser io use supports it..

**Comment by daanbreur** [comment]
# APPROVED, MERGED

**Pull request merged**